addons/addon-base-raas-ui/packages/base-raas-ui/src/helpers/api.js

```diff
@@ -58,7 +58,7 @@ function createAwsAccount(awsAccount) {
 
 // Note the accountUUID used here is the 'id' column in dbAwsAccounts table and 'id' attribute in AwsAccount.js, not AWS account id
 function updateAwsAccount(accountUUID, data) {
-  return httpApiPut(`api/aws-accounts/${accountUUID}/update-account`, { data });
+  return httpApiPut(`api/aws-accounts/${accountUUID}`, { data });
```

Comment:
What does this change do? Was it the wrong API endpoint before and the account wasn't getting updated?

Reply:
This change doesn't do anything functionality wise, it just changes the API call to be consistent with all the other `update` functions (ex. indexes, users, studies)